src/modules/timepix3/handlers/PixelEventHandler.h

```diff
@@ -118,7 +121,7 @@ class PixelEventHandler
    */
   PixelEventHandler(Counters &counters,
                     std::shared_ptr<Timepix3Geometry> geometry,
-                    EV44Serializer &serializer);
+                    EV44Serializer &serializer, const int &FrequencyHz = 14);
```

Comment:
I need to understand why this is necessary (maybe it becomes obvious further down in the PR diff)

Reply:
The pixel event handler calculates the max TOF per pulse from the frequency in the config, and then it will discard those events how's are actually belong to the next pulse. But this is not the final design, probably I should handover here a nanosecond value instead of the integer